### DISCUSSION TITLE
ایجاد بخش skin in the game

### DISCUSSION THREAD

**Discussion by flydeoo** [Hameds/GO_SIYANAT_YOURSELF#1] [Ideas]
سلام @Hameds جان

الان توییتر رو نگاه کردم و "اینترنت فراموش نمیکند" رو دیدم. ایده ای که دارم : 
 اضافه کردن جدولی که اظهار نظر های مختلف مسئولان در مورد طرح صیانت رو به صورت موافقت یا مخالفت هر فرد نسبت به نسخه ی های مختلف طرح صیانت کاملا مشخص میکنه.
 
 | Name  |  ۱۳۹۷/۰۹/۱۱ | ۱۳۹۹/۰۴/۲۴ | ۱۴۰۰/۰۴/۲۶ | 
| ------------- | ------------- | ------------- | ------------- |
| Mr X  | + | + | + |
| Mr Y  | - | + | +

**Comment by Hameds** [comment]
این پیشنهاد بسیار خوبیه.
